### PR TITLE
Fix isolation between dynamic Objective-C classes in multiple frameworks

### PR DESCRIPTION
**Pull request by SvyatoslavScherbina**
Also add test and improve handling duplicates for `@ExportObjCClass`.

### CODE REVIEW

File: runtime/src/main/cpp/ObjCInterop.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+#ifndef RUNTIME_OBJCINTEROP_H
+#define RUNTIME_OBJCINTEROP_H
+
+#if KONAN_OBJC_INTEROP
+
+extern "C" const char* Kotlin_ObjCInterop_uniquePrefix;
```
Comment (olonho) [inline runtime/src/main/cpp/ObjCInterop.h]: Maybe function instead?

File: backend.native/tests/framework/multiple/multiple.swift [hunk 2]
```diff
@@ -36,14 +36,31 @@ func testIsolation() throws {
     try assertTrue(Second.RuntimeState().consumeChange())
 }
 
+func testIsolation2() throws {
+    try assertEquals(actual: FirstKt.getI1().getFortyTwo(), expected: 42)
+    try assertEquals(actual: SecondKt.getI2().getFortyTwo(), expected: 42)
+}
+
+func testIsolation3() throws {
+/*
```
Comment (olonho) [inline backend.native/tests/framework/multiple/multiple.swift]: Maybe `#if 0`?
